tensorflow/python/ops/array_ops.py

```diff
@@ -5871,3 +5871,55 @@ def repeat(input, repeats, axis=None, name=None):  # pylint: disable=redefined-b
     input = reshape(input, [-1])
     axis = 0
   return repeat_with_axis(input, repeats, axis, name)
+
+@tf_export(v1=["mask_fill"])
```

Comment:
```suggestion
@tf_export("mask_fill")
```

The `v1=[...` part makes the API available only as part of `tf.compat.v1` and TF1.15